src/openforms/config/tests/test_privacy_policy.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+
+from openforms.config.models import GlobalConfiguration
+
+
+class PrivacyPolicyTests(TestCase):
+    def test_render_privacy_label(self):
+        conf = GlobalConfiguration.get_solo()
+        conf.privacy_policy_label = "I read the {% privacy_policy %} and agree."
+        conf.privacy_policy_url = "http://test-privacy-policy.nl"
+        conf.save()
+
+        label = conf.render_privacy_policy_label()
+
+        self.assertEqual(
```

Comment:
use `self.assertHTMLEqual` (or something like that) which also parses and validates the HTML.